utils/clipboard/clipboard.go

```diff
@@ -0,0 +1,68 @@
+package clipboard
+
+import (
+	"crypto/sha256"
+	"crypto/subtle"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"os/exec"
+	"strconv"
+
+	otherClipboard "github.com/atotto/clipboard"
+)
+
+const (
+	clearClipboardCmd = "clear-clipboard"
+	hashLen           = 4 // in bytes
+)
+
+// CopyWithClearing copies the provided text to the system clipboard
+// If clearAfter is given and larger 0, the clipboard is automatically
+// cleared after the duration has passed
+func CopyWithClearing(text string, clearAfter Duration) error {
```

Comment:
What are your thoughts on renaming this to `Copy`? I think its much more concise and improves the `clipboard` package API.

Additionally, we should always be clearing the clipboard and given the `clearAfter Duration` param and function comment, it's pretty clear.